cmd/bridge/main.go

```diff
@@ -108,6 +108,11 @@ func main() {
 	fStatuspageID := fs.String("statuspage-id", "", "Unique ID assigned by statuspage.io page that provides status info.")
 	fDocumentationBaseURL := fs.String("documentation-base-url", "", "The base URL for documentation links.")
 
+	fAlermanagerPublicURL := fs.String("alermanager-public-url", "", "Public URL of the cluster's AlertManager server.")
```

Comment:
Should be `Alertmanager` instead of `AlertManager`

Reply:
I agree this one should be changed since it's public facing.